src/index.html

```diff
@@ -2,16 +2,193 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="inputForm"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldsetBlock">
+        <legend>Personal information</legend>
+        <div class="inputBlock">
+          <label for="surname">Surname:
+            <input
+              autocomplete="off"
+              id="surname"
+              type="text"
+              name="surname"
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="name">Name:
+            <input
+              autocomplete="off"
+              id="name"
+              type="text"
+              name="name"
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="age">How old are You?
+            <input
+              id="age"
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+
+        </div>
+
+        <div class="inputBlock">
+          <label for="birthDay">Full date of birth:
+            <input
+              id="birthDay"
+              type="date"
+              name="birthDay"
+            >
+          </label>
+
+        </div>
+
+        <div class="inputBlock">
+          <label for="acceptAgree">I accept the term of the agreement
+            <input
+              id="acceptAgree"
+              type="checkbox"
+              name="acceptAgree"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldsetBlock">
+        <legend>Registration</legend>
+
+        <div class="inputBlock">
+          <label for="email">E-mail:
+            <input
+              id="email"
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="password">Password:
+            <input
+              id="password"
+              type="password"
+              name="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldsetBlock">
+        <legend>An interesting fact about you</legend>
+
+        <div class="inputBlock">
+          <label for="loveCats">Do you love cats?
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            >
+          </label> Yes
+
+          <input
+            type="radio"
+            name="loveCats"
+            value="no"
+          >
+        </label> No
+        </div>
```

Comment:
let's work on fixes here
- you don't need "Do you love cats?" in the label, label is connected to radio option
- move your `No` and `Yes` texts inside labels so they would be a part of input
- you don't have opening tag for second label
- you don't need for attribute if you don't have id
```suggestion
        <div class="inputBlock">
          Do you love cats?
          <label>
            <input
              type="radio"
              name="loveCats"
              value="yes"
            >
            Yes
          </label>
          
          <label>
	          <input
	            type="radio"
	            name="loveCats"
	            value="no"
	          >
	          No
          </label> 
        </div>
```